.github/workflows/build.yml

```diff
@@ -100,7 +103,7 @@ jobs:
       matrix:
         arch: [arm64, amd64]
         hv: [xen, kvm, kubevirt]
-        platform: ["generic"]
+        platform: ["generic", "evaluation"]
```

Comment:
If you are adding the `[amd64,kvm,evaluation]` to include, then you don't need to put in platform, otherwise is going to generate for all possible combinations of `(arch,hv,platform)`...